core/include/usBundleEvent.h

```diff
@@ -108,7 +108,32 @@ class US_Core_EXPORT BundleEvent
      * <p>
      * The bundle has been removed from the Framework.
      */
-    UNINSTALLED
+    UNINSTALLED,
```

Comment:
You are right, it will also be consistent with the rest of the code.

Now, these enum identifiers are not consistent in style with the one from `Bundle::State` and should get a prefix (e.g. `BUNDLE_`) if we are not going to use scoped enums everywhere.